src/domain/event/Event.tsx

```diff
@@ -69,7 +75,8 @@ const Event: FunctionComponent = () => {
     .filter((c) => c.childId === params.childId)
     .pop()
     ?.eventIds.some((e) => e === params.eventId);
-  if (isRegistered) {
+  if (isRegistered && !past) {
+    console.log('isRegistered');
```

Comment:
Left in on  purpose?

Reply:
Thanks!! Good spot, a proof that it is important to have someone else looking over my shoulder!